.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  build_docs:
+    docker:
+      # see: https://circleci.com/docs/2.0/circleci-images/#python
+      - image: circleci/python:3.7-stretch
+    steps:
+      # Get our data and merge with upstream
+      - run: sudo apt-get update
+      - checkout
+
+      - restore_cache:
+          keys:
+            - cache-pip
+
+      - run: |
+          pip install numpy==1.17.4
+          pip install --user .[docs,testing]
+
+      - save_cache:
+          key: cache-pip
+          paths:
+            - ~/.cache/pip
+
+      # Build the docs
+      - run:
+          name: Build docs to store
+          # nit-picky mode, turn warnings into errors,
+          # but do not stop the build on errors (so we can still inspect the doc artifacts)
+          # this needs to re re-instated when doc-revamp merged
+          # command: |
+          #   sphinx-build -b html -nW --keep-going -d docs/_build/doctrees docs/source docs/_build/html
+          command: |
+            sphinx-build -b html -n --keep-going -d docs/_build/doctrees docs/source docs/_build/html
+          # environment:
+            # READTHEDOCS: 'True'
+            # RUN_APIDOC: 'False'
```

Comment:
Should this be enabled? If not, best to remove it. But I guess we need `RUN_APIDOC = True` for the full build to succeed at some point no?

Reply:
I think that's the default anyways, setting it to 'False' will disable it. No setting means it will run it.

Reply:
Yeh it's enabled by default, I just had these there for testing but can remove now if you want?

Reply:
I don't think it really matters, fine with me to leave this.

Reply:
fixed in 9bebcb7